CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 SECURITY:
 
 * secrets/transit: fix a regression that was honoring nonces provided in non-convergent modes during encryption. [[GH-22852](https://github.com/hashicorp/vault/pull/22852)]
+* auth/cert: compare public keys of trusted non-CA certificates with incoming
```

Comment:
@mickael-hc this will go out in RC3, I'm pretty sure. I don't think based on when it merged that it was included in RC2.

Reply:
Thank you ! Fixed in https://github.com/hashicorp/vault/pull/25775/commits/4511ae59f6f8d92eb209f70bd2fa0ef820f15014